src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -320,7 +318,6 @@ private static AnalyzeJobOutput analyzeRepos(String outputPath, List<RepoConfigu
                             + MESSAGE_START_ANALYSIS, configToAnalyze.getLocation(), configToAnalyze.getBranch()));
             try {
                 GitRevParse.assertBranchExists(configToAnalyze, FileUtil.getBareRepoPath(configToAnalyze));
-                GitLsTree.validateFilePaths(configToAnalyze, FileUtil.getBareRepoPath(configToAnalyze));
```

Comment:
This code is specifically run for Windows, I don't think it should be removed?

Reply:
At the moment, running RepoSense on Windows against a repo with a single file that contains special characters will cause the entire repo to be rejected by this line (#1616) as it first views all filenames in the repo and throws an error. 

I think a better approach will be to skip the file which is what will be done after this line of code is removed as the rest of the repository is analyzed as per normal. Unless the intention is to disallow RepoSense to work on any such repositories. I think the fix is relatively simple as it can still be cloned but the specific file cannot be worked on directly.

Reply:
This was introduced in #564 but I could not find a reason why it was decided to just skip the repository entirely. @fzdy1914 Do you have any opinion about this?

Reply:
@chan-j-d Your suggestion makes sense. Just wondering, will the analysis of other files be normal given that this file is wrong?

Reply:
@fzdy1914 the other files are not affected. In general, files with filenames containing special characters are not processed properly for authorship and skipped regardless of OS and I am still investigating. So if someone narrows the time window to just a single commit and compares the number of lines to the authorship lines, there will be a mismatch. But with regards to the analysis for the remainder files, they are still accurate at least from my experience. 

For the run of RepoSense highlighted in #1616 that highlights this problem, I ran the same command with the master branch RepoSense on WSL and compared the report generated to one from this pull request branch ran on Windows, The summary and authorship json files are exactly the same (tested with `cmp`) so it is at least consistent.

Reply:
Hi @fzdy1914, @dcshzj, would like to ask if there is any further action suggested for this PR?

Reply:
@chan-j-d I think there are no other comments, feel free to just skip the file and maybe add a warning in the report to show the user that the file was skipped due to invalid characters.

Reply:
Thanks for the update @dcshzj. Would like to clarify what you mean by a warning in the report. The current analysis already handles it appropriately and outputs a warning as per the image in the issue for Windows systems. Or are you referring to a warning on the report when opened in a web browser similar to that of when a repo fails?

Reply:
> The current analysis already handles it appropriately and outputs a warning as per the image in the issue for Windows systems.

Yup, I was referring to this. If it is already handled then it's all good.